src/D2L.Bmx/Okta/OktaSessionStorage.cs

```diff
@@ -7,6 +7,12 @@ internal class OktaSessionStorage {
 
 	internal static void SaveSessions( List<OktaSessionCache> sessions ) {
 
+		if( !ConfigFileExists() ) {
+			Console.WriteLine( "No config file found. Your session will not be saved. " +
```

Comment:
nit - I think it's clearer this way
```suggestion
			Console.WriteLine( "No config file found. Your Okta session will not be cached. " +
```